apps/studio/components/interfaces/Organization/TeamSettings/InviteMemberButton.tsx

```diff
@@ -79,6 +87,8 @@ export const InviteMemberButton = () => {
     { enabled: canReadSubscriptions }
   )
   const currentPlan = subscription?.plan
+  const isOptedIntoProjectLevelPermissions =
+    useIsOptedIntoProjectLevelPermissions(slug as string) && currentPlan?.id === 'enterprise'
```

Comment:
this ok? @joshenlim

looks like logic change

Reply:
yup! this one's okay - a user should only be opted into project level permissions based on this 2 conditions
- whether is opted in via flag
- and if the plan is enterprise